Runtime/Data/Type/FloatRange.cs

```diff
@@ -21,6 +21,11 @@ public class FloatRange
         [DocumentedByXml]
         public float maximum;
 
+        /// <summary>
+        /// Shorthand for writing FloatRange(0f, 0f).

```

Comment:
```suggestion
        /// Shorthand for writing <c>FloatRange(0f, 0f)</c>.
```
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/code-inline